packages/cms/pages/components/dashboard/layout/avatar-context-menu.tsx

```diff
@@ -33,11 +44,11 @@ export const AvatarContextMenu: React.FunctionComponent = () => {
 
   const items = [
     <EuiContextMenuItem
```

Comment:
Is there a way to make this a link ? That way when we hover over it, the browser recognises it as a link ? I think we can use the Link component from react router